zh/restore-data-using-tidb-lightning.md

```diff
@@ -98,18 +100,104 @@ tikv-importer 可以在一个现有的 TiDB 集群上启用，或者在新建 Ti
+            type: Opaque
+            stringData:
+              rclone.conf: |
+              [s3]
+              type = s3
+              provider = AWS
+              env_auth = false
+              access_key_id =
+              secret_access_key =
+              region = us-east-1
+              [ceph]
+              type = s3
+              provider = Ceph
+              env_auth = false
+              access_key_id = <my-access-key>
+              secret_access_key = <my-secret-key>
+              endpoint = <ceph-object-store-endpoint>
+              region = :default-placement
+              [gcs]
+              type = google cloud storage
+              # 该服务账号必须被授予 Storage Object Viewer 角色。
+              # 该内容可以通过 `cat <service-account-file.json> | jq -c .` 命令获取。
+              service_account_credentials = <service-account-json-file-content>
+            ```
+
             使用你的实际配置替换上述配置中的占位符，并将该文件存储为 `secret.yaml`。然后通过 `kubectl apply -f secret.yaml -n <namespace>` 命令创建该 `Secret`。
 
         3. 将 `dataSource.remote.storageClassName` 设置为 Kubernetes 集群中现有的一个存储类型。
 
 2. 部署 TiDB Lightning
 
+    + 使用 AWS S3 AccessKey 和 SecretKey 权限授予方式 或者 Ceph，GCS 作为存储后端时：
+
     {{< copyable "shell-regular" >}}
 
     ```shell
     helm install pingcap/tidb-lightning --name=<tidb-lightning-release-name> --namespace=<namespace> --set failFast=true -f tidb-lightning-values.yaml --version=<chart-version>
     ```
 
+    + 使用 AWS S3 IAM 绑定 Pod 的授权方式时，需要做以下步骤：
+
+    1. 创建 IAM 角色：
+
+    可以参考 [AWS 官方文档](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html)来为账号创建一个 IAM 角色，并且通过 [AWS 官方文档](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html) 为 IAM 角色赋予需要的权限。由于 `Backup` 需要访问 AWS 的 S3 存储，所以这里给 IAM 赋予了 `AmazonS3FullAccess` 的权限。
+
+    2. 修改 tidb-lightning-values.yaml, 找到字段 `annotations`，增加 annotataion `iam.amazonaws.com/role : arn:aws:iam::123456789012:role/user`
+
+    3. 部署 Tidb-Lightning：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm install pingcap/tidb-lightning --name=<tidb-lightning-release-name> --namespace=<namespace> --set failFast=true -f tidb-lightning-values.yaml --version=<chart-version>
+    ```
+
+> **注意：**
+>
+> `arn:aws:iam::123456789012:role/user` 为步骤 1 中创建的 IAM 角色。
+
+    + 使用 AWS S3 IAM 绑定 ServiceAccount 授权方式时：
+
+    1. 在集群上为服务帐户启用 IAM 角色：
+
+        可以参考 [AWS 官方文档](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) 开启所在的 EKS 集群的 IAM 角色授权。
+
+    2. 创建 IAM 角色：
+
+        可以参考 [AWS 官方文档](https://docs.aws.amazon.com/eks/latest/userguide/create-service-account-iam-policy-and-role.html)创建一个 IAM 角色，为角色赋予 `AmazonS3FullAccess` 的权限，并且编辑角色的 `Trust relationships`。
+
+    3. 绑定 IAM 到 ServiceAccount 资源上：
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        kubectl annotate sa <servie-account> -n eks.amazonaws.com/role-arn=arn:aws:iam::123456789012:role/user
+        ```
+    4. 部署 Tidb-Lightning：
```

Comment:
We have to set sa in values.yaml

Reply:
use set-string to fill wthe  servcieaccount field